tools/srcgenerator.py

```diff
@@ -69,6 +69,7 @@ def generate_jerry_core(output_dir, verbose=False):
         '--input={}/include/jerryscript.h'.format(JERRY_CORE),
         '--output={}/jerryscript.h'.format(output_dir),
         '--remove-include=config.h',
+        '--push-include={}/include/jerryscript-debugger-transport.h'.format(JERRY_CORE),
```

Comment:
At the moment `jerryscript-debugger-transport.h` is not included from `core/include/jerryscript.h` (though `jerryscript-debugger.h` is, for example). This causes an issue when building with `ENABLE_ALL_IN_ONE_SOURCE`, because `jerryscript-debugger-transport.c` will be added to the generated `jerryscript.c`, but `jerryscript-debugger-transport.h` won't be present in the `jerryscript.h`.

Would you prefer `jerryscript-debugger-transport.h` to be listed in `core/include/jerryscript.h`, like the regular debugger? Or maybe you have an idea for a better solution?